app/bundles/PointBundle/Event/PointBuilderEvent.php

```diff
@@ -77,6 +77,12 @@ public function addAction($key, array $action)
             $action
         );
 
+        //Support for old way with callback and new event based system
+        //Could be removed after all events will be refactored to events. The key 'eventName' will be mandatory and 'callback' will be removed.
+        if (!array_key_exists('callback', $action) && !array_key_exists('eventName', $action)) {
```

Comment:
Since it's m2-to-m3 I think we should drop support for callbacks now and refactor to events